src/features/presentation/layouts/presentation/presentation.layout.html

```diff
@@ -0,0 +1,141 @@
+<div style="position: relative">
+  <img src="../../../../../../assets/img/presentation/fond-landing.svg" class="img-fluid" alt="image_fond_landing" />
+  <div class="d-flex row justify-content-center align-items-center my-5" style="position: absolute; top: 0">
+    <div class="col-6 d-flex my-5 align-items-center justify-content-center">
+      <img
+        src="../../../../../../assets/img/presentation/personnages_sur_carte.svg"
+        class="img-fluid"
+        alt="image_personnages_sur_carte"
+        style="width: 80%" />
+    </div>
+    <div class="col-6 d-flex flex-column justify-content-center">
+      <h1 class="h1 fw-bold text-secondary mb-4" style="width: 95%">
+        Bienvenue sur la cartographie nationale des lieux d'inclusion numérique
+      </h1>
+      <p class="text-secondary mb-4" style="width: 90%">
+        Cette plateforme vise à faciliter l’identification des structures de médiation numérique ainsi que l’orientation des
+        usagers finaux. Avant d'accéder à la cartographie, vous pourrez en quelques clics exprimer les besoins de votre
+        bénéficiaire et trouver les structures qui y répondent le mieux.
+        <br />
+      </p>
+      <div class="w-40">
+        <a [routerLink]="['orientation']" class="btn text-primary bg-blue-light" style="width: auto; border-radius: 0%">
```

Comment:
```suggestion
        <a [routerLink]="['orientation']" class="btn text-primary bg-blue-light rounded-0" style="width: auto">
```
À quoi sert `width: auto` ici ?